qiskit/transpiler/preset_passmanagers/level2.py

```diff
@@ -118,6 +118,7 @@ def _vf2_match_not_found(property_set):
             call_limit=int(5e6),  # Set call limit to ~10 sec with rustworkx 0.10.2
             properties=backend_properties,
             target=target,
+            max_trials=int(25000),  # Limits layout scoring to < 6 sec on ~400 qubit devices
```

Comment:
We don't need the cast here, the reason we did on `call_limit` is because `5e6` is a float by default. But in this case you're defining it as an int, so the cast isn't needed.

```suggestion
            max_trials=25000,  # Limits layout scoring to < 6 sec on ~400 qubit devices
```

Reply:
Heh, yeah I lost my mind and left it in after deciding to change to ints.